java/memory/memory-core/pom.xml

```diff
@@ -54,6 +53,30 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>error-prone-jdk11+</id>
```

Comment:
how is this profile related to ErrorProne?

Reply:
This profile is really the "default" build profile for JDK9+. It doesn't relate to error-prone specifically. Would it be better to have a separate profile with the same activation?

Reply:
Ah, then just rename it so it's clear - the "error-prone" profiles we have are meant for running the ErrorProne static analyzer

Reply:
Changed the profile to be jdk11+.